amundsen_application/static/js/components/common/Flag/index.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+// TODO: Use css-modules instead of 'import'
+import './styles.scss';
+
+interface FlagProps {
+  text: string;
+  style?: string;
+}
+
+const Flag: React.SFC<FlagProps> = ({ text, style }) => {
+  return (
+    <span className={`flag label ${style}`}>{text.toUpperCase()}</span>
```

Comment:
Maybe put a TODO here when we upgrade to bootstrap 4. 'label' should be changed to 'badge'